### ISSUE TITLE
v1.19.4 on Fedora 33 - gating test failure [1]

### ISSUE THREAD

**Issue by lsm5** [containers/buildah#3010]
**Description**

From @nalind on IRC: 139: the <<< operator appends a newline to the "php:7.2" it's given as an argument, the tr call replaces that final newline with a "-", podman load tries to make an image name out of "php:7.2-", and fails because that's not valid

**Describe the results you received:**

See: https://osci-jenkins-1.ci.fedoraproject.org/job/fedora-ci/job/dist-git-pipeline/job/master/19549/testReport/(root)/tests/root_test/

**Output of `rpm -q buildah` or `apt list buildah`:**

```
buildah-1.19.4-2.fc33.x86_64
```

**Output of `buildah version`:**

```
Version:         1.19.4
Go Version:      go1.15.7
Image Spec:      1.0.1-dev
Runtime Spec:    1.0.2-dev
CNI Spec:        0.4.0
libcni Version:  
image Version:   5.10.1
Git Commit:      
Built:           Wed Dec 31 19:00:00 1969
OS/Arch:         linux/amd64
```

rpms involved in gating test: https://bodhi.fedoraproject.org/updates/FEDORA-2021-fb466fb623

```
buildah-1.19.4-2.fc33
containernetworking-plugins-0.9.1-2.fc33
containers-common-1-4.fc33
podman-3.0.0-1.fc33  
skopeo-1.2.1-1.fc33
```

**Comment by lsm5** [comment]
@edsantiago can I close this one? This was waived in bodhi iirc.

**Comment by edsantiago** [comment]
I never saw this PR, never saw the failure. The failure log shows a ton of other errors that have since been fixed; and 1.19.4 is pretty old anyway. So although I don't understand the 7.2 failure itself, I haven't seen it in production, so I'm OK with closing.
